### ISSUE TITLE
404 Returns on Internet Explorer/Edge

### ISSUE THREAD

**Issue by gadams999** [rufuspollock/s3-bucket-listing#53]
Hi,
Using the BUCKET_URL and S3BL_IGNORE_PATH set to false, I can navigate to a CNAME'ed url and have it load the initial listing from index.html. However, all links return 404's. I have the error document on the bucket's website hosting set to index.html also, but it's not working. Chrome and Firefox work properly.

Has anyone seen or debugged this issue?

**Comment by rufuspollock** [comment]
@gadams999 I haven't seen this but i don't have IE :grimacing: Let us know how you get on and if anyone else can reproduce please say.

**Comment by gadams999** [comment]
@rgrp I'm pretty sure I found it. The initial `index.html` would load (200 message) but the navigation would fail when the file should be used for 404 messages. A quick search pulled up a _feature_ of IE and Edge. Error pages less than 512 bytes would not be used but the _friendly name_ would be.

In my index.html I added a comment in the <head> to set the value to at least 512 bytes and now the listing is working fine, testing in both IE and Edge on Windows 10. Want me to do a pull request to add a padding comment to the index.html file and reference in the readme.md?

Side note, while testing, I noticed that rgrp.github.io was CNAME'd to a fastly. net address that was failing to resolve to the actual list.js file. So my testing was done with local hosting of the javascript.
